### PR TITLE
CHAIN-660: allow numbers in env.yml files; update testnet fixtures; u…

### PR DESCRIPTION
**Pull request by bholzman**
…se full chain name in ui config

### CODE REVIEW

**Review by antonyshchenko — APPROVED** [review]
Looks great!

File: scripts/envs/demo.yml [hunk 1]
```diff
@@ -10,19 +10,19 @@ environment:
...
     MAKER_FEE_RATE: "0.0005"
     TAKER_FEE_RATE: "0.001"
     FAUCET_MODE: AllSymbols
-    TRADE_SETTLEMENT_BATCH_MIN_TRADES: "10"
-    TRADE_SETTLEMENT_BATCH_MAX_WAIT_TIME_MS: "30000"
-    WITHDRAWAL_SETTLEMENT_BATCH_MIN_WITHDRAWALS: "5"
-    WITHDRAWAL_SETTLEMENT_BATCH_MAX_WAIT_MS: "10000"
+    TRADE_SETTLEMENT_BATCH_MIN_TRADES: 10
+    TRADE_SETTLEMENT_BATCH_MAX_WAIT_TIME_MS: 30000
+    WITHDRAWAL_SETTLEMENT_BATCH_MIN_WITHDRAWALS: 5
```
Comment (bflood366) [inline scripts/envs/demo.yml:21]: Ah I had removed the word SETTLEMENT from these 2 WITHDRAWAL_ params in one of my PRs but had not updated the name in the yaml file. The name are `EVM_WITHDRAWAL_BATCH_MIN_WITHDRAWALS` and `EVM_WITHDRAWAL_BATCH_MAX_WAIT_MS`

**Review by bflood366 — APPROVED** [review]
looks great

File: scripts/envs/demo.yml (hunk 1 above)
Comment (bholzman) [reply scripts/envs/demo.yml]: Thanks, will change

**Pull request merged**